app/configurator/components/chart-configurator.tsx

```diff
@@ -231,6 +223,21 @@ const useEnsurePossibleFilters = ({
 
         const oldFilters = getChartConfigFilters(chartConfig.cubes, cube.iri);
 
+        // Replace resolved values with potential dynamic max values to not
+        // override the dynamic max value with the resolved value
+        for (const [key, value] of Object.entries(oldFilters)) {
+          if (
+            value.type === "single" &&
+            isDynamicMaxValue(value.value) &&
+            filters[key]
+          ) {
+            filters[key] = {
+              type: "single",
+              value: `${value.value}`,
+            };
+          }
+        }
```

Comment:
Not sure I understand this 😓 Can you give me an example, or add a test 😁 ?

Reply:
I'll add a test, but just to explain: possible filters coming from the backend do not return `VISUALIZE_MAX_VALUE`, but rather an actual date (e.g. 2023), so if we didn't replace it back with `VISUALIZE_MAX_VALUE`, we would potentially trigger an update that shouldn't happen (as in fact, the pinned date didn't change, but is still equal to `VISUALIZE_MAX_VALUE`). Maybe it would make sense to already do this replacement on the backend side? 🤔